index.css

```diff
@@ -0,0 +1,204 @@
+html, body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    color: rgb(75, 75, 75);
+}
+
+.navPane{
+    width: 30vw;
+    height: 100vh;
+    position: fixed;
+    margin: 0;
+    background-image: linear-gradient(to top, #fff1eb 0%, #ace0f9 100%);
+    padding-left: 3vw;
+    padding-top: 10vh;
+}
+
+@keyframes slideRight {
+    from {
+        color: rgba(75, 75, 75, 0.0);
+        left: -2vw;
+    }
+    to {
+        color: rgba(75, 75, 75, 1.0);
+        left: 0;
+    }
+}
```

Comment:
nice work trying out animations.  The animations seem to be off though, when I viewed them in the github html preview though.